src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  line-height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.8);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  display: flex;
+  padding-left: 50px;
```

```suggestion
  margin-left: 50px;
```
 Тут краще було б використати margin